gulpfile.js

```diff
@@ -526,7 +370,7 @@ const { launch, connect } = require('hadouken-js-adapter');
 			done();
 		},
 		launchApplication: done => {
-			if (!isNodeVersionValid()) {
+			if (!isNodeVersionValid(MAX_NODE_VERSION)) {
```

Comment:
Good catch.

Reply:
actually not a catch. previous MAX_NODE_VERSION was a global defined in gulpfile. I moved the isNodeVersionValid function out of gulpfile.